hw5/Task_2.ts

```diff
@@ -0,0 +1,42 @@
+//Task 2
+// Напишите функцию которая в качестве аргумента принимает в себя сумму кредита,
+// который хочет получить клиент и верните результат переплаты по кредиту:
+// процентная ставка в год — 17%,
+//  количество лет — 5.
+//  Мы пишем функцию для хорошего банка, поэтому сумма кредита не увеличивается.
+
+let percent: number = 0.17;
+let years: number = 5;
+let sumOfCreditOverPay;
+
+function credit(sumOfCredit: number) {
+    sumOfCreditOverPay = sumOfCredit - (sumOfCredit * percent) * years
+    return sumOfCreditOverPay
+}
+
+console.log(credit(1000))
```

Comment:
Please adjust the formula, because current result is incorrect, only for first year payment will be 170 = 1000*0,17

![image](https://user-images.githubusercontent.com/47811445/216768246-084ffc45-b518-4e5d-8893-da5692bcd251.png)